UPGRADES.md

```diff
@@ -1,3 +1,23 @@
+## 5.6 to 5.7
+
+No breaking changes were introduced. However, going forward when processing a `LtiOidcLogin`, it is recommended to use the new `getRedirectUrl()` method:
+
+```php
+// Do this:
+$redirect = $oidcLogin->getRedirectUrl($request);
```

Comment:
Both `getRedirectUrl()` and `doOidcLoginRedirect()` take `$launchUrl` as it's first argument and `$request` as the second. Could you update these examples to reflect that?